20181216_VJAI_MLMeetup_#13.md

```diff
@@ -0,0 +1,49 @@
+
+Hạn cuối đăng ký là **23:59 thứ 3 - 11/12/2018**.
+
+https://goo.gl/forms/pjMCFs7gDayq7sW93
+
+Ai không kịp đăng ký trước thời hạn trên sẽ không được vào toà nhà nên ngoài ấn Going thì các bạn **nhớ điền vào form** trên nhé.
+
+Lần này ban tổ chức đã sắp xếp phòng rộng hơn nhưng cũng chỉ chứa được 50 người nên nếu số người đăng ký nhiều hơn, ban tổ chức sẽ áp dụng hình thức Lottery (như đã thông báo trên facebook group). Thêm nữa là việc vào toà nhà không dễ dàng nên các bạn tính toán thời gian cẩn thận để đến đúng giờ nhé. (TBC)
+
+Hẹn gặp lại cả nhà vào buổi sắp tới!
+
+Nội dung chương trình:
+---
+### Phạm Quang Khang
+AI Solutions Architect
+
+### Pay more attention to Attention mechanism
+Since proposed in 2017, Attention has been evolved and become one of the most attended mechanism in fields like computer vision, nature language processing and even in recommendation system. Taking machine translation problem as an example, this talk will walk through the intuition of this mechanism as well as its evolution from encode-decoder attention to self-attention and lastly, Transformer (Attention is all you need: rank 4th most popular paper of all time on Arxiv Sanity), which removed completely conventional RNNs architecture.
+
+### References
+1. Bahdanau et al,. Neural Machine Translation by jointly learning to align and translate
+2. Thang Luong et al,. Effective Approaches to Attention-based Neural Machine Translation
+3. Lin et al,. A structured self-attentive sentence embedding Vaswani et al,.
+4. Multilple authors, Attention is all you need
+---
+
+### Nguyễn Phước Tất Đạt
+AI Research Engineer (BizReach, Inc.)
+
+### Machine can do reading comprehension test for you
+Reading comprehension, or question answering, is one of the most important and challenging natural language processing (NLP) tasks, in which a concrete answer is produced with respect to a question on an input text. This task is hard for even human when the input texts are complex, and it is used to test language proficiency of readers.
+
+With the advancement of Deep Learning (DL) in many domains, NLP researchers are seeking for various ways to apply DL techniques to NLP tasks, including reading comprehension task. In this trend, a robust deep neural network architecture is designed to capture hierarchical representations of language, which then maps a question and a context as inputs directly to the appropriate answer as output.
+
+In this talk, we will explore some neural network architectures for reading comprehension task which give state-of-the-art results even better than human performance. At the end of the talk, let's give machine an input text, then you ask it answers.
```

Comment:
`ask` nghĩa là hỏi mà nhỉ. `ask a answer` nghe hơi lạ (ask a question thì quen hơn) :D

Reply:
`you ask it answers`, not `ask a answer`. Đây là 2 phrases, 2 subjects và 2 verbs.

Reply:
à ra vậy :))
search google thì nó thêm dấu phẩy vào giữa `you ask, it answers`